python/thirdparty/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+set -ex
+
+APACHE_ARROW_VERSION=release-8.0.0
+
+# Build apache arrow
+function build_arrow {
+    git clone git@github.com:apache/arrow
+    pushd arrow
+    git checkout ${APACHE_ARROW_VERSION}
+    git submodule update --init
+    pushd python
+    pip install -r requirements-build.txt
+
+    export OPENSSL_ROOT_DIR=/opt/homebrew/opt/openssl@1.1
```

Comment:
what's the diff between `openssl@1.1` and `openssl@3`

Reply:
Not sure. This is the one installed on my system. Would be great to make CMake does not require these exports on Mac.